### PR TITLE
Super-Low Prio: Fix typo in documentation

### PR DESCRIPTION
**Pull request by area1211**
Take a look at [https://oracle.github.io/node-oracledb/doc/api.html#-4261-executemany-sql-statement](url)

Fix typo '4.2.6.1' to '4.2.7.1'

### CODE REVIEW

**Comment by cjbj** [comment]
@area1211 Thanks for the PR.  

Our dev process starts with commits in a private repo, where we can do leading-edge development for Oracle DB features that are not yet public.  We then replicate commits to GitHub at appropriate intervals.  I have updated our repo with the typo fix and have pushed to GitHub on the current development branch dev-4.0.  I also pushed to the gh-pages branch, which is associated with the home page https://oracle.github.io/node-oracledb/doc/api.html#executemanysqlparam  

The GitHub master branch is currently for node-oracledb 3.1.  The master branch is unlikely to be updated until we are ready to release node-oracledb 4.0. We made an exception for the patch allowing builds with Node.js 12.

In summary, PRs are very welcome so we can accept contributions and review them, but we don't do normal GitHub merges since the changes need to go to our private repo first.

**Comment by cjbj** [comment]
Closing this since it has been fixed.  I look forward to more PRs from you.
